WooCommerce/Classes/ViewRelated/Products/Media/WordPressMediaLibraryImagePickerViewController.xib

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
yea it's currently empty since it's embedding a child view controller fullscreen. I didn't really like creating an empty xib, but I thought I'd follow the current UI creation convention to always include a xib (also in case we want to customize this screen in the future). I can't wait for SwiftUI 😬